dashboard/django/collector.py

```diff
@@ -210,45 +271,26 @@ def _fetch_broker_stats(cluster, broker_host_port, timestamp):
                 topic.localThroughputIn = topic.msgThroughputIn - replicationThroughputIn
                 topic.localThroughputOut = topic.msgThroughputIn - replicationThroughputOut
 
+    # For all DB providers we have to insert one by one
+    # to be able to retrieve the PK of the newly inserted records
+    for bundle in db_bundles:
+        bundle.save()
 
-    if connection.vendor == 'postgresql':
```

Comment:
Since the Pulsar dashboard docker container already comes with Postgres, we should not remove this code path. 

Having the bulk insert significantly improves the insertion speedup which is critical when monitoring a larger number of topics.

Reply:
I didn't find the same bulk update method as bulk create for different objects, so I split the update into a single action.   https://github.com/apache/pulsar/pull/3236/commits/d94a07da370d3c71692bababc2be70fff3cc7641